tests/utils/call-cli.js

```diff
@@ -2,8 +2,15 @@ const execa = require('execa')
 
 const cliPath = require('./cli-path')
 
+const CLI_TIMEOUT = 3e5
+
 const callCli = async function (args, execOptions) {
-  const { stdout } = await execa(cliPath, args, { windowsHide: true, windowsVerbatimArguments: true, ...execOptions })
+  const { stdout } = await execa(cliPath, args, {
+    windowsHide: true,
+    windowsVerbatimArguments: true,
```

Comment:
Is this needed? [The default behavior](https://github.com/sindresorhus/execa#windowsverbatimarguments) might be better at handling Windows.

Reply:
Don't think it is. Removed in https://github.com/netlify/cli/pull/1727/commits/130ee481e20c00e3d4c45f9fd3ceea59414b28e7